.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+* @cyberark/community-and-integrations-team @conjurinc/community-and-integrations-team @conjurdemos/community-and-integrations-team
```

Comment:
Should we care about the errors ?

Reply:
IDK but this is in all the repos ;)

Reply:
I might be wrong but think the codeowners here need only be `@cyberark/...`

Reply:
I would think so too but this predates me. It's just copied from the other repos. I'd love to remove it if someone can confirm it's ok to

Reply:
Yeah I think the template includes all of them with the intention that the unnecessary ones get deleted.